configs/networks.json

```diff
@@ -1,16 +1,46 @@
 [
   {
-    "label": "Starknet Mainnet",
     "name": "mainnet",
-    "url": "https://starknet.preview.apibara.org",
-    "indexer": true,
-    "description": "The Starknet mainnet"
+    "url": "starknetid-mainnet.starknet.a5a.ch",
```

Comment:
Why all these changes ? What is the contractAddress for ? The new url doesn't work, etc...
It would be better to revert this imo